thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionTaskRunner.java

```diff
@@ -138,6 +147,19 @@ private void runTask(DateTime windowStart, DateTime windowEnd) throws JobExecuti
     storeData(adOutputContext);
   }
 
+  private double calculateTrafficImpact (MetricTimeSeries metricTraffic, MergedAnomalyResultDTO mergedAnomaly) {
+    Set<Long> timestamps = metricTraffic.getTimeWindowSet();
+    double avgTraffic = 0.0;
+    int count = 0;
+    for (long timestamp : timestamps) {
+      if (timestamp >= mergedAnomaly.getStartTime() && timestamp <= mergedAnomaly.getEndTime()) {
+        avgTraffic += metricTraffic.get(timestamp, mergedAnomaly.getMetric()).doubleValue();
+        count++;
+      }
+    }
+    avgTraffic /= count;
+    return (mergedAnomaly.getAvgCurrentVal() - mergedAnomaly.getAvgBaselineVal()) / avgTraffic;
```

Comment:
Yes. I assume the metric value without dimension and filter as the traffic itself. I will add comments describing this.